actionpack/lib/action_dispatch/middleware/session/cache_store.rb

```diff
@@ -9,13 +9,16 @@ module Session
     # of time.
     #
     # ==== Options
-    # * <tt>cache</tt>         - The cache to use. If it is not specified, <tt>Rails.cache</tt> will be used.
-    # * <tt>expire_after</tt>  - The length of time a session will be stored before automatically expiring.
+    # * <tt>cache</tt>            - The cache to use. If it is not specified, <tt>Rails.cache</tt> will be used.
+    # * <tt>expire_after</tt>     - The length of time a session will be stored before automatically expiring.
     #   By default, the <tt>:expires_in</tt> option of the cache is used.
+    # * <tt>cache_key_prefix</tt> - The string prepended to the session id to form the cache key.
+    #   Defaults to <tt>_session_id:</tt>
     class CacheStore < AbstractStore
       def initialize(app, options = {})
         @cache = options[:cache] || Rails.cache
         options[:expire_after] ||= @cache.options[:expires_in]
+        @cache_key_prefix = options[:cache_key_prefix] || '_session_id:'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.